Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
I guess for now, except for the ones that we have designated as sticky at the top, I think running running the remainder through poet_lint should be Good Enough.